autonat.go

```diff
@@ -12,10 +12,13 @@ import (
 	ma "github.com/multiformats/go-multiaddr"
 )
 
+// NATStatus is the state of NAT as detected by the ambient service.
 type NATStatus int
 
 const (
-	// NAT status is unknown
+	// NAT status is unknown; this means that the ambient serice has not been
+	// able to decide the presence of NAT in the most recent attempt to test
```

Comment:
s/decide/detect

Reply:
Maybe determine then. Decide is just an odd way of phrasing this sentence.

Reply:
it's a very precise term in computability.

Reply:
Ok. Could you point towards any resources that define it in that context? I haven't come across this before. Cheers 🙂